microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: devtools
-ms.date: 05/15/2024
+ms.date: 06/12/2024
 ---
 # Crash analyzer tool
 
-Use the **Crash analyzer** tool to analyze and quickly diagnose crashes of your production web app.  In the **Crash analyzer** tool, you can input a JavaScript stack trace, such as for non-fatal JavaScript exceptions, and then have your source maps applied to the stack trace so that you can debug faster.  The **Crash analyzer** tool enables you to take a minified JavaScript stack trace and rapidly work backwards to determine what lines of code caused the error.
+Use the **Crash analyzer** tool to analyze and quickly diagnose crashes of your web app in production.  In the **Crash analyzer** tool, you can input a JavaScript stack trace that you collected in production and then have your source maps applied to the stack trace so that you can debug faster.  The **Crash analyzer** tool enables you to take a minified JavaScript stack trace and rapidly work backwards to determine what lines of code caused the error.
 
+<!-- TODO: Redo screenshot -->
 ![The Crash analyzer tool](./index-images/crash-analyzer.png)
-<!-- todo: Create new demo page & use it in this article.
-Or, use page/file:
-https://devblogs.microsoft.com/oldnewthing/
-devtools://devtools/bundled/panels/elements/elements.js
-devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js
 
-show stacktrace content in DevTools
--->
+When a web app crashes or hangs unexpectedly, this can result in data loss and a poor experience for your users.  Collecting information about the crashes that your users experience is helpful to diagnose and fix the root cause of the crashes. The **Crash analyzer** tool helps you map minified stack traces collected in production back to the original source code, so that you can quickly identify the root cause of the crashes.
 
-When a web app crashes or hangs unexpectedly, this can result in data loss and a poor user experience.  The **Crash analyzer** tool helps you identify the cause of these crashes.  Use the **Crash analyzer** tool to analyze the memory state of your web app at the time of a crash.
+There are two steps to analyze minified production stack traces in the **Crash analyzer** tool:
 
-As a web developer, you collect stack traces from customers in production, and then use the **Crash analyzer** tool to "beautify" the stack traces in DevTools.  DevTools only does the analysis of the stack traces, not the collecting of stack traces.
+1. First, you collect specially-formatted stack traces from your web app in production.
```

Comment:
fyi should be no hyphen after "ly" - an exception.  i'll probably push commit to fix.  
@captainbrosset

Reply:
fixed 3x